src/pages/Loginpage/index.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react';
+import { Form } from 'react-bootstrap';
+import { Formik, ErrorMessage } from 'formik';
+import * as Yup from 'yup';
+import styled from 'styled-components';
+import { Redirect } from 'react-router-dom';
+import CustomButton from 'components/Button';
+
+
+const OuterContainer = styled.div`
+    background-color:#1E90FF;
+    position: fixed;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    top: 0;
+`;
+
+const InnerContainer = styled.div`
+    background-color: white;
+    width: 50%;
+    height: 70%;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 80px;
+    padding-top: 70px;
+    padding-left: 30px;
+    padding-right: 30px;
+    border-radius: 20px;
+`;
+
+const Header = styled.h1`
+    text-align: center;
+`;
+
+const FormWrapper = styled.div`
+    margin-top: 30px;
+`;
+
+const BtnWrapper = styled.div`
+    margin-top: 50px;
+`;
+
+const ErrorWrapper = styled.div`
```

Comment:
keep style components in separate files

Reply:
Created a new file for styles Loginpage.styles.ts